generic-styles/index.less

```diff
@@ -133,6 +133,11 @@ body {
       border-bottom: 1px solid #dcdcdc;
       text-transform: uppercase;
     }
+
+    .os-coreq-exercises .os-problem-container,
+    .os-coreq-exercises .os-problem-container > p {
```

Comment:
This will display inline all paragraphs
https://rex-web-issue-1297-8sgcl2jj0kt.herokuapp.com/books/college-algebra-corequisite-support/pages/1-3-radicals-and-rational-exponents
Before:
![image](https://user-images.githubusercontent.com/31478212/93878140-dfbc8c00-fcd9-11ea-9ff5-cf262357819a.png)

After:
![image](https://user-images.githubusercontent.com/31478212/93878178-eb0fb780-fcd9-11ea-8d3d-627e2c2774f8.png)